### ISSUE TITLE
Add additional features

### ISSUE THREAD

**Issue by saikrishnarallabandi** [saikrishnarallabandi/falkon#7]
Large fluctuations due to less training data. May be better to add additional features that we know will help the task.

**Comment by saikrishnarallabandi** [comment]
A version of this logic is implemented in the following script:

https://github.com/saikrishnarallabandi/falkon/blob/master/tasks/speech/self_assessed_affect/baseline/local/seqmodels/baseline_attention.py

This has to be stress tested against different tasks and scenarios. 

Talk to me :)

**Comment by saikrishnarallabandi** [comment]
Some resources:

https://datascience.stackexchange.com/questions/31113/validation-showing-huge-fluctuations-what-could-be-the-cause

http://cs231n.stanford.edu/reports/2017/pdfs/300.pdf

https://www.quora.com/What-might-be-the-cause-of-the-many-fluctuations-in-validation-accuracy-during-training-of-my-deep-neural-network

https://www.datacamp.com/community/tutorials/towards-preventing-overfitting-regularization

**Comment by saikrishnarallabandi** [comment]
Does this fluctuation have something to do with CDF? 

Figure 1 in  https://arxiv.org/pdf/1803.00047.pdf

**Comment by saikrishnarallabandi** [comment]
@peterwu99

**Comment by saikrishnarallabandi** [comment]
Pretraining on Google level data might help (?) @peterwu99
